skyfield/reify.py

```diff
@@ -0,0 +1,89 @@
+from functools import update_wrapper
+
+# Source: 
+# https://docs.pylonsproject.org/projects/pyramid/en/latest/api/decorator.html
+# Copyright and license information below.
+
+class reify(object):
```

Comment:
I'd been thinking of adding my own version of this, and I think I have a branch somewhere — if I can find it, I'll try merging so that you can use that code instead.

Reply:
All right! I've merged in and pushed an update so that `@reify` is now available. Could you try rebasing and seeing if it works correctly for your case as well? Thanks!